style/main.css

```diff
@@ -36,6 +36,7 @@ h1.title {
     font-weight: bold;
     content: "Star Wars"; }
 
+
```

Comment:
CSS is generated by [main.scss](https://github.com/0x0800/2048-STARWARS/blob/gh-pages/style/main.scss), check [CONTRIBUTING.md](https://github.com/0x0800/2048-STARWARS/blob/gh-pages/CONTRIBUTING.md) about this.